src/main/resources/application.properties

```diff
@@ -8,5 +8,9 @@ spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql=true
 #debug=true
 
+security.jwt.secret-key=JtsAJGrGhaTUNl5SQ/HaimPSll0nNSHZe7RiMvAupAw=
+# 1h in millisecond
+security.jwt.expiration-time=3600000
```

Comment:
better indicate naming to be clear the value, instead of using comment, like 
`security.jwt.expiration-time-in-ms`